site/src/pages/TasksPage/TasksPage.stories.tsx

```diff
@@ -261,6 +266,17 @@ export const CreateTaskSuccessfully: Story = {
 			await userEvent.click(submitButton);
 		});
 
+		await step("Uses latest template version", () => {
+			expect(API.experimental.createTask).toHaveBeenCalledWith(
```

Comment:
Correct me if I'm wrong, but since this is a story not a unit test, you shouldn't assert API calls on that level?

Can we convert it to jest test instead?